src/Microsoft.Health.Fhir.Core.UnitTests/Features/Operations/Export/CancelExportRequestHandlerTests.cs

```diff
@@ -19,12 +19,15 @@
 using Microsoft.Health.Fhir.Core.Features.Persistence;
 using Microsoft.Health.Fhir.Core.Features.Security.Authorization;
 using Microsoft.Health.Fhir.Core.Messages.Export;
+using Microsoft.Health.Fhir.Tests.Common;
 using Microsoft.Health.Test.Utilities;
 using NSubstitute;
 using Xunit;
 
 namespace Microsoft.Health.Fhir.Core.UnitTests.Features.Operations.Export
 {
+    [Trait("Traits.OwningTeam", OwningTeam.Fhir)]
```

Comment:
General questions:
1. Do we need to do this with Unit and Integration tests since they only run in our OSS pipeline?
2. How can we create a rule to ensure the attributes are applied in new test files?

Reply:
1. Yes, the intention is classifying all types of tests. We want to have it ready for any possible filter in pipelines executions in the future. Those ownership tags will also help identifying the owning team during troubleshooting. 

2. I'm checking if I can create something with stylecop or code analysis.

Reply:
Ok. At the end, I've created a set of tests with the category AssemblyValidation. 
Those tests are using reflection and checking if every single testing class has a Category and an OwningTeam. 

I'm identifying Test Classes scanning all classes with methods decorated with [Fact] or [Theory] attributes.